### ISSUE TITLE
Using capacitor jitsi meet  am able join meeting but other participants not displying

### ISSUE THREAD

**Issue by jyothimarkal** [calvinckho/capacitor-jitsi-meet#65]
Hi,

Am trying to do online classes using capacitor jitsi meet in ionic and angular , in android device am able join meeting but other participants not displaying..plz help me 

capacitor 3
and 
 implementation ('org.jitsi.react:jitsi-meet-sdk:3.10.2') { transitive = true }

**Comment by calvinckho** [comment]
can you describe in more detail what do you mean by "other participants not displaying"? On my android, I am able to join a meeting, and the other participants are displaying correctly.

![](https://d2z4pehxidbzz4.cloudfront.net/users/596dc94782e2eb4512a320b6/16389753025311ut/image.jpg)

**Comment by jyothimarkal** [comment]
Hi,
Thank you for the response. As your showing in the screenshot their we can see 2 participants but in mine am not able to see other participants
![ea174873-4447-4bd6-9377-47be988bea73](https://user-images.githubusercontent.com/7098985/145381887-ca99aada-20f3-4906-9df4-c9a8d22768a3.jpg)
![WhatsApp Image 2021-12-09 at 4 11 24 PM](https://user-images.githubusercontent.com/7098985/145381892-4ab1c2c2-f476-4729-b870-b3568e155127.jpeg)

so I have doubt like while adding meetings from the server-side do we have to create any JWT token and send that token from each student how is going to join that meeting.

**Comment by calvinckho** [comment]
Can you test your conference as follow?

1. With a mobile phone that is running your app plugin, join a jitsi room name of you choice, e.g. myappjitsiroom123. Use the default jitsi video bridge endpoint. After you have joined, you should see only 1 participant (the mobile phone user) in the room. 

2. Next, on a computer browser, go to https://meet.jit.si/myappjitsiroom123 and join the room. 

3. Once that is done, on your mobile phone, you should now see 2 participants: the mobile user and the web browser user.

**Comment by jyothimarkal** [comment]
Sorry for the late reply am hanging on to another task, now I will chk and get back soon.

**Comment by jyothimarkal** [comment]
I am trying all the steps as you told still am facing the same problem,

web browser
![image](https://user-images.githubusercontent.com/7098985/145584008-47873a02-2ff3-45e0-991d-ba7a074277db.png)


mobile
![Screenshot_2021-12-10-19-13-54-44 (1)](https://user-images.githubusercontent.com/7098985/145584945-ed1bfa97-9510-4c50-ab8b-9ce365fe0799.png)


But when I use scripting to connect jitsi meet ( in ionic angular ) it will show the participants and works fine but the problem is I want to show this in android app

**Comment by jyothimarkal** [comment]
Hi ,
 am waiting for your reply.
My code:

import { Jitsi } from 'capacitor-jitsi-meet';

async join(meeting_id:string)
{
  console.log("join : ",meeting_id);
   const result: any = await Jitsi.joinConference({
    roomName: meeting_id, // room identifier for the conference
    url: 'https://meet.jit.si', // endpoint of the Jitsi Meet video bridge
    **token:  this.token, // jwt authentication token**
    displayName:  this.name, // user's display name
    email: 'string', // user's email
    avatarURL: 'string', // user's avatar url
    channelLastN: '10', // last N participants allowed to join
    startWithAudioMuted: true, // start with audio muted
    startWithVideoMuted: true ,// start with video muted
    chatEnabled: false, // enable Chat feature
    inviteEnabled: false, // enable Invitation feature
    callIntegrationEnabled: true // enable call integration (CallKit on iOS, ConnectionService on Android)
  });
  console.log("join status", result.success);

  window.addEventListener('onConferenceJoined', () => {
     // do things here
     console.log("addEventListener ", result.success);
  });
  window.addEventListener('onConferenceLeft', () => {
     // do things here
  });

}

here i have doubt wat is JWT token ?

**Comment by calvinckho** [comment]
Sorry I really don't know why you can't see other participants. Maybe others have insights?

For detailed info about jwt tokens, see this issue: #26

**Comment by jyothimarkal** [comment]
Okay Thank you for your response.

**Comment by calvinckho** [comment]
Since we are unable to reproduce your issue, I am closing this for now. If you have new information which can help us reproduce your issue, we can reopen this issue.

**Comment by calvinckho** [comment]
@gigallo my advice would be to run android deployments on Android Studio, and compare the logcat console outputs. 

If the conference is joined successfully, the log should be like this one:

`2022-02-17 16:37:11.990 1330-13065/com.restvo.app D/JitsiMeetSDK: ExternalAPI Sending event: CONFERENCE_JOINED with data: { NativeMap: {"url":"https://meet.jit.si/61ef084df7c52b6b5ffd5c56"} }
2022-02-17 16:37:11.992 1330-1330/com.restvo.app I/JitsiMeetSDK: Conference joined: {url=https://meet.jit.si/61ef084df7c52b6b5ffd5c56}
2022-02-17 16:37:11.992 1330-13065/com.restvo.app D/JitsiMeetSDK: JitsiConnectionService reportConnectedOutgoingCall FFBD0398-E11F-47CD-82F6-2C3E3CF36537
2022-02-17 16:37:11.993 1330-13065/com.restvo.app D/JitsiMeetSDK: JitsiConnectionService onStateChanged: ACTIVE FFBD0398-E11F-47CD-82F6-2C3E3CF36537
2022-02-17 16:37:11.996 7452-9932/? D/ConnectionServiceWrapper: ConnectionService -> Telecom[com.restvo.app/org.jitsi.meet.sdk.ConnectionService]: setActive TC@37_1
2022-02-17 16:37:11.996 7452-9932/? I/Telecom: CallsManager : setCallState DIALING -> ACTIVE, call: [TC@37, DIALING, com.restvo.app/org.jitsi.meet.sdk.ConnectionService, null, ATR, childs(0), has_parent(false), unknown, unknown [Capabilities:] [Properties: PROPERTY_SELF_MANAGED]]
2022-02-17 16:37:11.999 7452-9932/? I/SecCallsManagerListener: onCallStateChanged DIALING -> ACTIVE, call: [TC@37, ACTIVE, com.restvo.app/org.jitsi.meet.sdk.ConnectionService, null, ATR, childs(0), has_parent(false), unknown, unknown [Capabilities:] [Properties: PROPERTY_SELF_MANAGED]]
2022-02-17 16:37:11.999 7452-9932/? I/MinuteMinder: currentCall = [TC@37, ACTIVE, com.restvo.app/org.jitsi.meet.sdk.ConnectionService, null, ATR, childs(0), has_parent(false), unknown, unknown [Capabilities:] [Properties: PROPERTY_SELF_MANAGED]]
2022-02-17 16:37:12.008 1330-13070/com.restvo.app I/JitsiMeetSDK: AudioMode Update audio route for mode: 2
2022-02-17 16:37:12.008 7452-9932/? D/InternetPermissionPopup: call : [TC@37, ACTIVE, com.restvo.app/org.jitsi.meet.sdk.ConnectionService, null, ATR, childs(0), has_parent(false), unknown, unknown [Capabilities:] [Properties: PROPERTY_SELF_MANAGED]], oldState : 3, newState : 5
2022-02-17 16:37:12.013 1330-13065/com.restvo.app D/JitsiMeetSDK: [JitsiConference.js] P2P? peerCount: 0, hasBotPeer: false => false
2022-02-17 16:37:12.016 1330-13070/com.restvo.app I/JitsiMeetSDK: AudioMode Selected audio device: SPEAKER
2022-02-17 16:37:12.018 1330-13070/com.restvo.app I/JitsiMeetSDK: AudioMode Updating audio device list
2022-02-17 16:37:12.039 7452-9932/? D/ConnectionServiceWrapper: ConnectionService -> Telecom[com.restvo.app/org.jitsi.meet.sdk.ConnectionService]: setAudioRoute TC@37_1 SPEAKER
2022-02-17 16:37:12.040 1330-1330/com.restvo.app D/JitsiMeetSDK: JitsiMeetView: onConferenceJoined`

**Comment by gigallo** [comment]
We don't know why but it seems a problem related to the signed apk app downloaded from the store. Signed apk app and unsigned apk app coming from android studio are not affected
